Automattic/wp-calypso
# Education Forum: Update /help courses content text

Related to https://github.com/Automattic/Customer-Education/issues/182

## Proposed Changes

Part of the work to remove/move away from the Education forum.

* **Task 3**: Remove `,and become part of its community` from `Enroll in a course taught by WordPress experts, and become part of its community.`

Before:

![CleanShot 2024-04-05 at 20 22 23@2x](https://github.com/Automattic/wp-calypso/assets/10482592/97da2628-1d4b-4b2f-8468-79950c3fac36)


After:

![CleanShot 2024-04-05 at 20 23 15@2x](https://github.com/Automattic/wp-calypso/assets/10482592/6e97ebc4-5435-4d15-a4d0-f620e879c6e5)


## Testing Instructions



* Checkout branch or use the live link.
* Navigate to `/help`, scroll down to the More Resources section, and verify the Courses content has been updated.

## Pre-merge Checklist
